src/Symfony/Component/Messenger/Transport/RedisExt/Connection.php

```diff
@@ -311,4 +379,14 @@ public function cleanup(): void
         $this->connection->del($this->stream);
         $this->connection->del($this->queue);
     }
+
+    public function getStream(): string
+    {
+        return $this->stream;
+    }
+
+    public function getGroup(): string
+    {
+        return $this->group;
+    }
```

Comment:
We don't expand the public API in patches, especially when it is only for testing purpose.
Can you revert and use hardcoded values in tests?

Reply:
@chalasr I first did hardcode it but @alexander-schranz pointed out to me that the DSN for the redis server isn't fixed and must be provided as an env parameter. Any idea how to accomplish this?

Reply:
@toooni I suggest to use reflection/closure binding for accessing the private property

Reply:
Thought because the `Connection` class is `@internal` its ok to extend it for tests.

Reply:
done 👍